xml/System.Diagnostics.Eventing.Reader/EventLogReader.xml

```diff
@@ -464,16 +464,52 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Reads the next event that is returned from the event query in this object.</summary>
-        <returns>Returns an <see cref="T:System.Diagnostics.Eventing.Reader.EventRecord" /> object.</returns>
+        <summary>Reads and returns the next event from the event query results - or null if the end of the results has been reached.</summary>
+        <returns>Returns an <see cref="T:System.Diagnostics.Eventing.Reader.EventRecord" /> object - or a null reference.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Examples  
- For example code using this class, see [How To: Query for Events](https://docs.microsoft.com/previous-versions/bb671200(v=vs.90)).  
-  
+          <para>
+            This overload is equivalent to calling <see cref="M:System.Diagnostics.Eventing.Reader.EventLogReader.ReadEvent(System.TimeSpan)" /> with an <see cref="M:System.Threading.Timeout.InfiniteTimeSpan" /> timeout parameter argument.
+          </para>
+          <para>
+            Each time this method is called the reader will advance to the next event record in its query results or the log file being read.
+            When the reader reaches the end of the query results or log file the method will return null on its next call, but subsequent calls will throw an InvalidOperationException.
+          </para>
+          <para>
+            The returned EventRecord objects implement IDisposable and should be disposed of appropriately. To avoid leaking resources consider copying desired data from the EventRecord object into a custom class or tuple and then immediately disposing of the returned EventRecord before advancing to the next logged event.
```

Comment:
```suggestion
            The returned EventRecord objects implement IDisposable and should be disposed of appropriately. To avoid leaking resources, consider copying desired data from the EventRecord object into a custom class or tuple and then immediately disposing of the returned EventRecord before advancing to the next logged event. The following code shows an example.
```